public/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    "extends": "airbnb",
+    "env": {
+        "browser": true,
+        "node": true,
+        "jasmine": true
+    },
+    "plugins": [
+        "react"
+    ],
+    "parser": "babel-eslint",
+    "rules": {
+        "indent": [0, 4, {"SwitchCase": 1}],
```

Comment:
Why disable indent and jsx-indent?

Reply:
because i had problem with props indent.
eslint required to use this syntax:
<SomeComponent prop1={value}
props2={value}
props3={value}
props4={value} 
/>
instead this:
<SomeComponent prop1={value}
                               props2={value}
                               props3={value}
                               props4={value} 
/>
Because of this, i temporary disabled jsx indent

Reply:
because i had problem with props indent.
eslint required to use this syntax:
`<SomeComponent prop1={value}
props2={value}
props3={value}
props4={value} 
/>`
instead this:
`<SomeComponent prop1={value}
                               props2={value}
                               props3={value}
                               props4={value} 
/>`
Because of this, i temporary disabled jsx indent

Reply:
I disabled indent because in .jsx it poorly works. For example, i could't indent text to 4 spaces. When i made 4 spaces eslint showed me, that i made 6 spaces. 

I disabled jsx indent in last step of refactoring because i was't satisfied with props indent.

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/ba75df40f0a73cc74ec3c0b1d624e338d6a7718b